doc/en/datetime.txt

```diff
@@ -402,6 +402,52 @@
 
 /* $DOC$
    $AUTHOR$
+      2017 Pete D. <pete_westg@yahoo.gr>
+   $TEMPLATE$
+      Function
+   $NAME$
+      hb_Week()
+   $CATEGORY$
+      API
+   $SUBCATEGORY$
+      Date/Time
+   $ONELINER$
+      Return the week number of year.
+   $SYNTAX$
+      hb_Week( <dDate>, [@<nYear>], [@<nDayOfWeek>] ) --> nWeek
+   $ARGUMENTS$
+      <dDate> Any valid date expression.
+
+		<nYear>, <nDayOfWeek> Optional parameters to obtain the year and day number of week.
+   $RETURNS$
+      <nWeek> The ordinal week number (of the year) into which falls the given <dDate>.
+   $DESCRIPTION$
+      This function returns the week number of year for the given <dDate>.
+		The returned value is an ISO 8601 compliant week number.
+      Optionally, can also obtained the year and week day number
+		into <nYear> and/or <nDayOfWeek> parameters if they are passed by reference.
+		Note: new function available after 2017-02-08 19:36 UTC+0100 commit,
+		      not found in earlier versions.
+   $EXAMPLES$
+		PROC Main
+		LOCAL nYear, nDayOfWeek
+      ? hb_Week( CtoD( "02/15/2017" ), @nYear, @nDayOfWeek ), nYear, nDayOfWeek
+		// output: 7, 2017, 3
```

Comment:
Please,
- remove the extra indent space from the `?` line
- use `0d20170215` instead of `CToD( ... )`, to make it work regardless of default date format
- run `hbmk2 -fixcase` on example code (though if the above line is updated anyway, it may be unnecessary)